{"name": "test_verify_login_with_valid_credentials", "status": "passed", "start": 1721882918015, "stop": 1721882919220, "uuid": "cec008fb-91b8-4dad-bb70-388743b4d09a", "historyId": "14f2072526db18090e3cb1e2c1ce29e3", "testCaseId": "14f2072526db18090e3cb1e2c1ce29e3", "fullName": "test.test_Login.TestLogin#test_verify_login_with_valid_credentials", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-9A3ESFK"}, {"name": "thread", "value": "12184-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_Login"}]}